exporters/otlp/metrics/src/test/java/io/opentelemetry/exporter/otlp/metrics/OtlpGrpcMetricExporterTest.java

```diff
@@ -70,11 +68,9 @@ void configTest() {
     Properties options = new Properties();
     options.put("otel.exporter.otlp.metric.timeout", "12");
     options.put("otel.exporter.otlp.insecure", "true");
-    OtlpGrpcMetricExporterBuilder config = OtlpGrpcMetricExporter.builder();
-    OtlpGrpcMetricExporterBuilder spy = Mockito.spy(config);
-    spy.readProperties(options);
-    verify(spy).setDeadlineMs(12);
-    verify(spy).setUseTls(false);
+    OtlpGrpcMetricExporterBuilder config = OtlpGrpcMetricExporter.builder().readProperties(options);
+    assertThat(config.getUseTls()).isFalse();
```

Comment:
rather than exposing a method, we can dig into the internals and keep the production code a bit cleaner. I can't say which is better or worse, but at least if we just dig into the internals, we've reduced a couple lines of production code and a discussion about naming those methods. ;)

Reply:
Nice